airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json

```diff
@@ -69,6 +69,64 @@
         "minimum": 0,
         "maximum": 90,
         "order": 6
+      },
+      "custom_reports": {
+        "title": "Custom Reports",
+        "description": "You can add your Custom Bing Ads report by creating one.",
+        "order": 7,
+        "type": "array",
+        "items": {
+          "title": "Custom Report Config",
+          "type": "object",
+          "properties": {
+            "name": {
+              "title": "Report Name",
+              "description": "The name of the custom report, this name would be used as stream name",
+              "type": "string",
+              "examples": [
+                "Account Performance",
+                "AdDynamicTextPerformanceReport",
+                "custom report"
+              ]
+            },
+            "reporting_object": {
+              "title": "Reporting Data Object",
+              "description": "The name of the the object derives from the ReportRequest object. You can find it in Bing Ads Api docs - Reporting API - Reporting Data Objects.",
+              "type": "string",
+              "examples": [
```

Comment:
Is it possible to `enum` these data objects for the user so we could have the dropdown selection?

Reply:
yes, all data objects are declared in [docs](https://learn.microsoft.com/en-us/advertising/reporting-service/reporting-data-objects?view=bingads-13), but it's more than 30, I'm not sure that is user-friendly to show such a big dropdown section

Reply:
Let's add them. It's fine to have the dropdown and quickly find the relative object right off, then go to the docs and try to understand what the object should be.